scripts/paypal/payment-reconciliator.ts

```diff
@@ -530,43 +467,108 @@ const findMissingPaypalTransactions = async (_, commander) => {
       totalPages = <number>response['totalPages'];
```

Comment:
Should we use the newly implemented `listPayPalTransactions`?

Reply:
Ideally, yes. In practice, I expect this script to become mostly useless with the new CRON job, which is why I didn't invest more time updating it.